docs/typescript-support.md

```diff
@@ -0,0 +1,67 @@
+# Typescript Support
+
+Styled-components has typescript definitions to allow the library to be
+used in any Typescript project.
+
+To use the library you can import it as you normally do with any TS dependency.
+
+```
+import styled from "styled-components";
```

Comment:
Could you remove the semicolon and change from double quotes to single quotes to have consistent style across the docs? Thanks! (this applies to all code snippets, not just this one)